grafast/grafserv/src/servers/whtatwg-node-server/index.ts

```diff
@@ -0,0 +1,115 @@
+import { createServerAdapter } from '@whatwg-node/server'
+
+import { GrafservBase } from "../../core/base.js";
+import type {
+  GrafservConfig,
+  RequestDigest,
+  Result,
+} from "../../interfaces.js";
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace Grafast {
+    interface RequestContext {
+        whatwg: {
+            request: Request
+        }
+    }
+  }
+}
+
+/** @experimental */
+export class WhatwgGrafserv extends GrafservBase {
+  protected whatwgRequestToGrafserv(
+    request: Request
+  ): RequestDigest {
+    const url = new URL(request.url);
+    return {
+      httpVersionMajor: 1,
+      httpVersionMinor: 0,
```

Comment:
1.0 seems unlikely, that's ancient! Are we not able to determine the HTTP version from a WHATWG server?

Reply:
Yeah I didn't like it either but couldn't find a way to get the HTTP version from a WHATWG Request object. Changed it to 1.1 but still hardcoded unfortunately